python/private/pypi/pip_args.bzl

```
@@ -0,0 +1,65 @@
+# Copyright 2023 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""
+
+def _get_extra_pip_args_for_platform(ctx, extra_pip_args):
+    """Returns the pip args for the given platform from the dict.
+
+    Args:
+        ctx: Current repository context.
+        extra_pip_args: (string keyed string list dict): Extra pip arguments.
+
+    Returns:
+        List of arguments for the current platform.
+    """
+
+    os = ctx.os.name.replace(" ", "")
+    arch = ctx.os.arch.replace(" ", "")
```

```suggestion
    os = ctx.os.name.replace(" ", "")
    arch = ctx.os.arch.replace(" ", "")
```

This is duplicating code from `repo_utils`. We already have code for getting `os` and `arch` labels that the rest of the code understands. See `host_platform` function in `parse_requirements.bzl` (if I remember correctly).